scripts/qa_crowbarsetup.sh

```diff
@@ -2304,6 +2330,15 @@ function oncontroller_testsetup()
     if ! ssh $vmip curl www3.zq1.de/test ; then
         complain 95 could not reach internet
     fi
+
+    # skip volume tests for docker
+    if [ "$image_name" == "cirros" ]; then
+        echo "RadosGW Tests: $radosgwret"
+        echo "Tempest: $tempestret"
+        echo "Volume in VM: skipped (not supported for docker)"
+        test $tempestret = 0 -a $radosgwret = 0 || exit 102
```

Comment:
I know, but the code is already there, the amount of added lines here is minimal